### PR TITLE
Bring back the UNUDHR

### PR DESCRIPTION
**Pull request by nateberkopec**
Chosen ethical framework cannot _deny_ any rights enumerated in the UNUDHR

### CODE REVIEW

**Review by CoralineAda — APPROVED** [review]
Love the clarification, thank you!

**Comment by nateberkopec** [comment]
Maybe we should clarify "does not infringe" tho - what would it mean for a framework to "infringe" the UDHR? Does the [Cairo Declaration](https://en.wikipedia.org/wiki/Cairo_Declaration_on_Human_Rights_in_Islam) "infringe"? What about the Bible? A party's political platform? etc etc.

File: content/definition.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Software is considered Ethical Source if it meets the following criteria:
 1. It is distributed with source code and can be freely used or combined with other software.
 1. It is developed or maintained in public view and welcoming of public contributions.
 1. Its community of contributors is governed by a code of conduct that is consistently and fairly enforced.
-1. It is distributed under a license that prohibits the software from being used any individual or organization which directly or indirectly facilitates, encourages, manipulates, coerces, or forces people to engage in behaviors that are in opposition to a specified framework of social good.
+1. It is distributed under a license that prohibits the software from being used by any individual or organization which directly or indirectly facilitates, encourages, manipulates, coerces, or forces people to engage in behaviors that are in opposition to a specified framework of social good, provided that the framework does not infringe on the rights enumerated in the United Nations Universal Declaration of Human Rights.
```
Comment (CoralineAda) [inline content/definition.md]: /infringe on/conflict with/ ?

Comment (nateberkopec) [reply content/definition.md]: is subsetting a conflict? e.g. if my ethical framework is for all rights in the UDHR, but not the freedom of religion, does it conflict?

Comment (nateberkopec) [reply content/definition.md]: I think the intention of the new wording of this clause is that this subsetting is _not_ an issue, but I just want to be clear.

Comment (nateberkopec) [reply content/definition.md]: If we're ok with subsetting, then I like `conflict` over `infringe`.

Comment (CoralineAda) [reply content/definition.md]: I like 'conflict' best?

Comment (CoralineAda) [inline content/definition.md]: "It is distributed under a license that prohibits its use "?

Comment (nateberkopec) [reply content/definition.md]: Not sure I understand this one, please expand?

Comment (elimisteve) [reply content/definition.md]: infringe upon?

Comment (CoralineAda) [reply content/definition.md]: Never mind :)

**Pull request merged**